trezor-crypto/include/TrezorCrypto/bip39_english.h

```diff
@@ -0,0 +1,367 @@
+/**
```

Comment:
there is two of the same content? `trezor-crypto/include/TrezorCrypto/slip39_wordlist.h` in this file there is a list as well

Reply:
Similar content, different purpose.  I think the slip39 has some helper index table, to map from keypad input to word.  Anyways, I have removed it.